readme.md

```diff
@@ -210,6 +210,7 @@
 - [Statusfy](https://github.com/bazzite/statusfy) - Status page system using Tailwind CSS.
 - [Todolist](https://github.com/guillaumebriday/todolist-frontend-vuejs) - To-do list application using Tailwind CSS.
 - [LeagueStats](https://github.com/vkaelin/LeagueStats) - Statistics website for League of Legends players.
+- [LearnAwesome](https://github.com/learn-awesome) - Humanity's universal learning map (built with Tailwind CSS)
```

Comment:
This needs to be at the bottom of the list. Please read [item 3 on the guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/contributing.md).